### DISCUSSION TITLE
Options for html fragments equivalent to "html_fragment" in RMarkdown

### DISCUSSION THREAD

**Discussion by suvendusamanta** [quarto-dev/quarto-cli#3529] [Q&A] [answered]
There is a output format "html_fragment" in RMarkdown.  Which generate partial HTML documents. They do not contain the standard header content that HTML documents do. I don't find such option in quarto documentation. Is there any equivalent format/option in quarto?

**Comment by rgaiacs** [comment]
@suvendusamanta thanks for the question. Can you provide details regarding your user case?

Some features from Quarto requires Javascript and CSS. Only the HTML will result in a poor rendering by the browser.

**Reply by suvendusamanta** [reply]
I don't have concrete use cases  right now but I want to innovate a personalized education platform. I am just exploring Quarto and comparing with parent project RMarkdown. Suppose I want to author a large number of atomic documents with mathematical equation and store into a database. Backend web server will serve those atomic documents  along with images, logic to construct pages etc. dynamically to the frontend application (probably a SPA ).  So the aesthetic part will be taken care by the  front end application.  Can I use quarto to generate barebone documents?

**Reply by rgaiacs** [reply, marked as answer]
You can use templates https://quarto.org/docs/journals/templates.html

**Reply by suvendusamanta** [reply]
👍 thank you.
